lib/Yield/Assembly/PTML.cpp

```diff
@@ -92,22 +94,27 @@ static std::string targetPath(const MetaAddress &Target,
   if (auto Iterator = Binary.Functions.find(Target);
       Iterator != Binary.Functions.end()) {
     // The target is a function
-    return "/function/" + str(Iterator->key());
+    return pipeline::serializedLocation(revng::pipes::ranks::Function,
```

Comment:
Put a `using namespace` to avoid `pipeline::` or make an alias just for this function.
We'll use this a lot, let's try to set a standard where things do not get super-verbose.

Reply:
Sure.

Reply:
Does this work? https://github.com/revng/revng/pull/197/commits/5451c5e3396623ebbeb488e9cfeb66d5291be88e.